plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/MapPart.java

```diff
@@ -32,7 +34,7 @@
  * @since 1.1.0
  * @version 1.3.0
  */
-public interface MapPart {
+public interface MapPart extends IWorkbenchPart {
```

Comment:
Thats great, with this change a MapPart can be a ViewPart or an EditorPart (Eclipse API)